src/main.js

```diff
@@ -40,18 +41,18 @@ class Master {
   }
 }
 
-// create a single instance of the p5Sound / master output for use within this sketch
-const p5sound = new Master();
+// create a single instance of the p5Sound main output for use within this sketch
+const p5sound = new Main();
 
 /**
  * Returns a number representing the master amplitude (volume) for sound
  * in this sketch.
  *
- * @method getMasterVolume
+ * @method outputVolume
```

Comment:
Should not it be *getOutputVolume*

Reply:
yes, thank you @endurance21 ! Just updated